app/search/api.py

```diff
@@ -13,9 +13,10 @@ def search_records(
             "size": results_per_page,
             "from": (page - 1) * results_per_page,
             "sort": sort,
-            "sortOrder": order,
         }
     )
+    # remove params having no values
+    params = {param: value for param, value in params.items() if value}
```

Comment:
Which params could this be?

Reply:
`sort=`, and any others coming through `params` parameter. Also when `sortOrder` is reinstated, it will be removed before passing them to the API.